server/constants/fields.js

```diff
@@ -47,6 +47,7 @@ const BASIC_FIELDS = [
   "medium",
   "imageOriginalSecret",
   "imageSecret",
+  "invno",
```

Comment:
We always want to retrieve the invno number as we need it when referencing objects in the DAMS